composer/algorithms/gradient_clipping/gradient_clipping.py

```diff
@@ -35,7 +35,18 @@ def apply_gradient_clipping(parameters: Union[torch.Tensor, Iterable[torch.Tenso
             to (for 'value'), what values to clip the gradient norms to (for 'norm'), and
             threshold by which if grad_norm / weight_norm is greater than this threshold then
             scale gradients by this threshold * (weight_norm / grad_norm) (for 'adaptive').
+        fsdp_enabled (bool): Bool of if the model is a FSDP model or not.
     """
+    if fsdp_enabled:
+        from torch.distributed.fsdp import FullyShardedDataParallel
+        assert clipping_type == 'norm'
```

Comment:
Clip by value is also supported out of the box because it is elementwise ;)

Reply:
oh hm.. by setting 'norm_type'='inf'?